ruby-challenges.rb

```diff
@@ -10,6 +10,17 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def includeLetter arr, char
+    return arr.filter do | k |
+        if k.include?(char)
+            k
+        end
+    end
+end
+
+p includeLetter beverages_array, letter_o
+p includeLetter beverages_array, letter_t
```

Comment:
👏 Excellent!  You actually don't need the return on line 14 since Ruby uses implicit returns, and no need for the if block when using .include.  Include automatically will return a boolean value and the return will be only values that are true in the filter.